extensions/wikia/Email/Controller/CategoryAddController.class.php

```diff
@@ -85,7 +85,7 @@ protected function afterSuccess() {
 	private function incrementEmailsSentCount() {
 		global $wgMemc;
 		$emailsSent = $wgMemc->get( $this->getTargetUserCacheKey() );
```

Comment:
Another solution is to cast to int which returns `false` or strings into `int(0)`.
```php
$emailsSent = ( int ) $wgMemc->get( $this->getTargetUserCacheKey() );
```